README.md

```diff
@@ -2,7 +2,21 @@
 
 ## Requerimientos
 
-Cualquier navegador que soporte ECMAScript 2015 (6th Edition, ECMA-262)
+Cualquier navegador que soporte ECMAScript 5 o superior
+
+### Navegadores soportados
+
+Por temas de seguridad y para proveer la mejor experiencia para tus clientes que usaran tu aplicación de pago,
+no damos soporte a navegadores que han dejado de recibir actualizaciones de seguridad y que actualmente 
+representan la minoría del trafico.
+
+Si bien hemos realizado pruebas completas y exitosas usando versiones de Microsoft Internet Explorer 10 y 11
```

Comment:
Yes, Eugenio made the test and it works.

Reply:
I did test the SDK on IE10 (under Win7), and it did work.
However I don't think it's a good idea to leave IE 10 as "kind-of-supported" (mentioning that we tried it and it works) since it will be us that will have give support for the SDK.
IE11 is fair game since it is supposed to be supported, though

Reply:
Ok i have removed the IE10 from the text